packages/app/src/ui/components/password-field/password-field.styled.tsx

```diff
@@ -19,12 +19,12 @@ export const FormInput = styled(InputComponent)`
 
 export const FieldWrapper = styled.div`
   position: relative;
-  margin-bottom: 1.4rem;
+  margin-bottom: 3.6rem;
 `
 
 export const IconWrapper = styled.div`
   position: absolute;
-  top: 0;
+  top: -1rem;
```

Comment:
why static value? What is the purpose of the change? Centering? What if the icon size will changes? I suggest changing the value to percent.